### ISSUE TITLE
Environment variables for configuring of default queue

### ISSUE THREAD

**Issue by Alexander-Makaryev** [IntelPython/dpctl#131]
It will be great to be able to select default queue outside of python scripts and internal logic.
with such feature we will be able to run tests of third party products (dpnp, daal4py, numba) on different devices/queues.
As a possible solution I suggest to use environment variable, for example ``DPCTL_DEFAULT_QUEUE``.

**Comment by PokhodenkoSA** [comment]
@diptorupd @Alexander-Makaryev 
`DPCTL_DEFAULT_QUEUE` should contain the same string as for `device_context(...)`?

**Comment by oleksandr-pavlyk** [comment]
It is already implemented in master, see #267 .

```
(idp) [18:37:51 nuc04 dpctl]$ SYCL_DEVICE_FILTER="opencl:cpu" \ 
    python -c "import dpctl; dpctl.get_current_queue().get_sycl_device().print_device_info()"
    Name            Intel(R) Core(TM) i7-10710U CPU @ 1.10GHz
    Driver version  2021.11.3.0.17_160000
    Vendor          Intel(R) Corporation
    Profile         FULL_PROFILE
    Device type     cpu
```
```
(idp) [18:38:04 nuc04 dpctl]$ SYCL_DEVICE_FILTER="opencl:gpu" \
    python -c "import dpctl; dpctl.get_current_queue().get_sycl_device().print_device_info()"
    Name            Intel(R) UHD Graphics [0x9bca]
    Driver version  21.11.19310
    Vendor          Intel(R) Corporation
    Profile         FULL_PROFILE
    Device type     gpu
```

```
(idp) [18:38:19 nuc04 dpctl]$ SYCL_DEVICE_FILTER="gpu" \
    python -c "import dpctl; dpctl.get_current_queue().get_sycl_device().print_device_info()"
    Name            Intel(R) UHD Graphics [0x9bca]
    Driver version  1.0.19310
    Vendor          Intel(R) Corporation
    Profile         FULL_PROFILE
    Device type     gpu
```
